test-e2e/android/app/src/main/java/com/mopro/mopro_app/MultiplierComponent.kt

```diff
@@ -23,8 +26,17 @@ fun MultiplierComponent() {
     var valid by remember { mutableStateOf("valid:") }
     var output by remember { mutableStateOf("output:") }
     var res by remember {
-        mutableStateOf<GenerateProofResult>(
-            GenerateProofResult(proof = ByteArray(size = 0), inputs = ByteArray(size = 0))
+        mutableStateOf(
+            CircomProofResult(
+                proof = CircomProof(
+                    a = G1(x = "", y = "", z = null),
+                    b = G2(x = listOf(), y = listOf(), z = null),
+                    c = G1(x = "", y = "", z = null),
+                    protocol = "",
+                    curve = ""
+                ),
+                inputs = listOf() // inputs 現在是 List<String>
```

Comment:
```suggestion
                inputs = listOf()
```

Reply:
fixed in [a0551c9](https://github.com/zkmopro/mopro/pull/372/commits/a0551c983b38309cd18d6e21617883a74b541b1f)